src/components/PauseVaultDirectorOffers.tsx

```diff
@@ -0,0 +1,146 @@
+import clsx from 'clsx';
+import { motion } from 'framer-motion';
+import {
+  LoadStatus,
+  usePublishedDatum,
+  usePublishedKeys,
+  WalletContext,
+} from 'lib/wallet';
+import { useContext, useEffect, useState } from 'react';
+
+interface ManagerGroupProps {
+  managerId: string;
+  onChecked: (checked: Record<string, boolean>) => void;
+}
+
+const ManagerGroup = ({ managerId, onChecked }: ManagerGroupProps) => {
+  const [checked, setChecked] = useState({
+    [`${managerId}: MakeVault`]: false,
```

Comment:
these strings don't look as meaningful as they are. consider create them with a function that explains their significance.
```suggestion
    [filterOffers(managerId, 'MakeVault')]: false,
```

Reply:
Done